deploy/main-existing-rg.bicep

```diff
@@ -0,0 +1,165 @@
+// name should not contain underscores
+
+@description('Prefix for created resources.')
+param appNamePrefix string
+
+@allowed([
```

Comment:
Generally speaking, which resource puts the location constraint in? What is only avail in these regions? Maybe put in a comment where that comes from